core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/EngineeringAllowanceManager.kt

```diff
@@ -73,7 +73,8 @@ class EngineeringAllowanceManager(private val graph: STDCMGraph) {
                     it.envelopeStartOffset + it.length.distance
                 )
             }
-        val pathProperties = makePathProps(graph.rawInfra, graph.blockInfra, blockRanges)
+        val routes = edges.lastOrNull()?.infraExplorer?.getExploredRoutes() ?: listOf()
```

Comment:
```suggestion
        val routes = edges.last().infraExplorer.getExploredRoutes()
```

This method wouldn't be called when the list is empty